### DISCUSSION TITLE
Google Drive Storage

### DISCUSSION THREAD

**Discussion by Braintelligence** [requarks/wiki#3948] [Help / Questions] [answered]
Hello everyone.

The docs say that you can use Google Drive as storage. I've set up Wiki.js via docker-compose but the Google Drive storage module is greyed out.

I can't find any further info on the web.

How do I set up Google Drive as storage?

**Comment by NGPixel** [comment, marked as answer]
That module is not available yet.

**Reply by Braintelligence** [reply]
I see, all sources I could find on the net said otherwise tho. Maybe this should be emphasized in the docs.

Is it on the roadmap?

**Reply by NGPixel** [reply]
The official site list the Google Drive storage module as coming in a future release:

https://js.wiki/modules#storage

**Reply by Braintelligence** [reply]
I meant this site for example: https://docs.requarks.io/storage

It literally says 
```
You can enable any of the storage modules below to do so.
```

And Google Drive is in the list.

**Reply by Braintelligence** [reply]
Just as a heads up: The docs still say that any of the mentioned storage modules can be enabled.

**Reply by NGPixel** [reply]
Constantly asking about it and pointing a phrase won't make it suddenly and magically work. I already answered your question that it is coming in a future release, please move on.

The phrase was merely a figure of speech to say that any number of modules can be enabled at the same time.
